content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -20,7 +20,7 @@ in their [JIRA project](https://issues.redhat.com/projects/DPP/summary) with the
 If your component repository is **not** in this organization:
 
 1. Invite `openshift-ci-robot` and `openshift-merge-robot` into your organization or add them as [collaborators](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository) for the repository.
-1. Contact a CI administrator to accept the invitation. Otherwise, This will happen automatically after some time.
+1. Contact a CI administrator to accept the invitation. Otherwise, This will happen automatically via the `	periodic-openshift-release-master-accept-invitations-ci-robot` and `periodic-openshift-release-master-accept-invitations-merge-robot` jobs in _no more_ than 4 hours.
```

Comment:
Can we completely omit the first sentence? The users have no way how to tell that our stuff broke, but I'd like to preempt pings for the happy case. If something breaks they will eventually disover it by "CI not working" which is not that hard to diagnose.

Reply:
Thats a good idea!